tests/api/Jalasoft.TeamUp.Projects.API.Tests/GetInvitationsByResumeIdTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Projects.API.Controllers;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Jalasoft.TeamUp.Projects.ProjectsException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Moq;
+    using Xunit;
+
+    public class GetInvitationsByResumeIdTest
+    {
+        private readonly Mock<IInvitationsService> mockService;
+        private readonly DefaultHttpContext mockHttpContext;
+        private readonly GetInvitationsByResumeId getInvitations;
+
+        public GetInvitationsByResumeIdTest()
+        {
+            this.mockService = new Mock<IInvitationsService>();
+            this.mockHttpContext = new DefaultHttpContext();
+            this.getInvitations = new GetInvitationsByResumeId(this.mockService.Object);
+        }
+
+        [Fact]
+        public void GetInvitationsByResumeId_Returns_OkObjectResult()
```

Comment:
please follow the naming convention the team is using https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices if there is any doubt for this please reach out to @lilian-delgado-jalafund  or @niver-copa-jalafund

Reply:
done